tools-v2/pkg/cli/command/curvebs/clean_recycle/clean_recycle.go

```diff
@@ -0,0 +1,218 @@
+/*
+ * Project: tools-v2
+ * Created Date: 2023-4-6
+ * Author: nanguanlin6@gmail.com
+ */
+
+package clean_recycle
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/nameserver2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+)
+
+const (
+	cleanRecycleBinExample = `$ curve bs clean-recycle --expiredtime=1h --recycleprefix=/test`
+	RECYCLEBINDIRNAME      = "RecycleBin"
+	RECYCLEBINDIR          = "/" + RECYCLEBINDIRNAME
+)
+
+// CleanRecycleCommand
+type CleanRecycleCommand struct {
+	basecmd.FinalCurveCmd
+	recyclePrefix string
+	expireTime    time.Duration
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*CleanRecycleCommand)(nil) // check interface
+
+// DeleteCertainFileRpc
+type DeleteCertainFileRpc struct {
+	Info      *basecmd.Rpc
+	Request   *nameserver2.DeleteFileRequest
+	mdsClient nameserver2.CurveFSServiceClient
+}
+
+var _ basecmd.RpcFunc = (*DeleteCertainFileRpc)(nil)
+
+func (gRpc *DeleteCertainFileRpc) NewRpcClient(cc grpc.ClientConnInterface) {
+	gRpc.mdsClient = nameserver2.NewCurveFSServiceClient(cc)
+}
+
+func (gRpc *DeleteCertainFileRpc) Stub_Func(ctx context.Context) (interface{}, error) {
+	return gRpc.mdsClient.DeleteFile(ctx, gRpc.Request)
+}
+
+// ListDirRpc
+type ListDirRpc struct {
+	Info          *basecmd.Rpc
+	Request       *nameserver2.ListDirRequest
+	curveFSClient nameserver2.CurveFSServiceClient
+}
+
+var _ basecmd.RpcFunc = (*ListDirRpc)(nil) // check interface
+
+func (lRpc *ListDirRpc) NewRpcClient(cc grpc.ClientConnInterface) {
+	lRpc.curveFSClient = nameserver2.NewCurveFSServiceClient(cc)
+}
+
+func (lRpc *ListDirRpc) Stub_Func(ctx context.Context) (interface{}, error) {
+	return lRpc.curveFSClient.ListDir(ctx, lRpc.Request)
+}
+
+// new CleanRecycleCommand function
+func NewCleanRecycleCommand() *cobra.Command {
+	crCmd := &CleanRecycleCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "clean-recycle",
+			Short:   "clean recycle bin",
+			Example: cleanRecycleBinExample,
+		},
+	}
+	return basecmd.NewFinalCurveCli(&crCmd.FinalCurveCmd, crCmd)
+}
+
+// method of CleanRecycleCommand struct
+func (crCmd *CleanRecycleCommand) Init(cmd *cobra.Command, args []string) error {
+	crCmd.recyclePrefix = config.GetBsRecyclePrefix(crCmd.Cmd)
```

Comment:
What is this for?

Reply:
> What is this for?

Delete the files in the recycle bin that its original filename has the prefix. I'm following the code in old tool of curve.